app/src/main/java/com/example/filemanager/AllFilesFragment.kt

```diff
@@ -28,16 +28,33 @@ class AllFilesFragment : Fragment(), ItemClickListener {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        setHasOptionsMenu(true)
```

Comment:
I guess you can remove this? Please make sure you remove code which is redundant and keep it clean.